cmd/gaia/testnets/README.md

```diff
@@ -353,6 +353,75 @@ gaiacli stake delegation \
   --chain-id=gaia-6002
 ```
 
+## Governance
+
+Governace is the process from which users in Cosmos can come to consensus on software updates, parameters of the mainnet or on custom proposals for independent sovereign zones/hubs.
+
+Governance is done through voting on proposals, which will be submitted by `Atom` holders on the mainnet. For more information about the governance process and how it works, please check out the Governance module [specification](https://github.com/cosmos/cosmos-sdk/tree/develop/docs/spec/governance).
+
+### Create a Governance proposal
+
+In order to create a governance proposal, you must submit an initial deposit along with the proposal details (_i.e_ `title`, `description` and `type`, which must be of value _Text_, _ParameterChange_ or _SoftwareUpgrade_):
```

Comment:
Should we note that parameter change and software upgrade proposals aren't implemented yet?

Reply:
yeah, I think we should